app/helpers/promotional_feature_helper.rb

```diff
@@ -0,0 +1,51 @@
+module PromotionalFeatureHelper
```

Comment:
This new file doesn't appear to have any tests - can some be added?

Reply:
I think @KludgeKML would be best placed for this as he wrote the code for this - but if he's busy I can try

Reply:
Added